File: 프로그래머스/2/12939. 최댓값과 최솟값/최댓값과 최솟값.java
import java.util.*;

class Solution {
    public String solution(String s) {
        
        StringBuilder sb = new StringBuilder();
        
        int min = Integer.MAX_VALUE;
        int max = Integer.MIN_VALUE;
        
        String[] arr = s.split(" ");
        
        for(int i=0; i<arr.length; i++){
            if(Integer.parseInt(arr[i]) < min){
                min = Integer.parseInt(arr[i]);
            }
            
            if(Integer.parseInt(arr[i]) > max){
                max = Integer.parseInt(arr[i]);
            }
            
        }
        sb.append(min).append(" ").append(max);
        return sb.toString();
        
        
    }
}

File: 프로그래머스/2/12939. 최댓값과 최솟값/최댓값과 최솟값.js
function solution(s) {
    
    let arr = s.split(' ');
    
    let min=arr[0],max=arr[0];
    
    
    arr.forEach(n => {
        max = Math.max(max,n)
        min = Math.min(min,n)
    })
    
    var answer = `${min} ${max}`;
    console.log(answer)
    
    
    return answer;
}